modules/hl7802/cellular_hl7802_api.c

```diff
@@ -525,6 +531,13 @@ static CellularError_t buildSocketConfig( CellularSocketHandle_t socketHandle,
                     socketHandle->socketProtocol ) );
         cellularStatus = CELLULAR_UNSUPPORTED;
     }
+    else if( strlen( socketHandle->remoteSocketAddress.ipAddress.ipAddress ) > CELLULAR_REMOTE_IP_ADDR_MAX_LENGTH )
```

Comment:
`remoteSocketAddress.ipAddress.ipAddress` - ipAddress twice seems odd. Do not worry if this PR did not introduce it.